administrator/language/de-DE/de-DE.ini

```diff
@@ -89,9 +89,12 @@ JINVALID_TOKEN="Die Anfrage wurde zurückgewiesen, da der Sicherheitstoken ungü
 JINVALID_TOKEN_NOTICE="Der Sicherheitstoken ist falsch. Die Anforderung wurde zurückgewiesen, um eine Sicherheitsverletzung zu verhindern. Aktualisieren Sie die Seite und versuchen Sie es erneut."
 JLOGIN="Anmelden"
 JLOGOUT="Abmelden"
+JMONTH="Monat"
+JMENU_MULTILANG_WARNING_MISSING_MODULES="Ein Administratormenü-Modul für <strong>%s</strong> existiert nicht. <br>Erstelle ein benutzerdefiniertes Administratormenü und -modul für jede Administratorsprache oder veröffentliche ein Menümodul, das auf Alle Sprachen eingestellt ist."
```

Comment:
Neutrale Sprache (kein Du und Sie). Nur in Ausnahmefällen „Sie“. Bindestriche nur bei zusammengesetzten Worten aus dem englischen.